nautobot/extras/tests/test_plugins.py

```diff
@@ -272,6 +282,93 @@ def test_registry_nav_menu_dict(self):
         )
 
 
+@skipIf(
+    "dummy_plugin" not in settings.PLUGINS,
+    "dummy_plugin not in settings.PLUGINS",
+)
+class PluginWebhookTest(SplinterTestCase):
+    """
+    This test case proves that plugins can use the webhook functions when making changes on a model.
+
+    Because webhooks use celery a class variable is set to True called `requires_celery`. This starts
+    a celery instance in a separate thread.
+    """
+
+    requires_celery = True
+
+    def setUp(self):
+        super().setUp()
+        self.url = f"http://localhost:{self.server_thread.port}" + reverse(
```

Comment:
Can you add some comments explaining why we're using `localhost` here instead of `self.live_server_url` as in other SplinterTestCases?

Reply:
Still a question I'd like to see addressed.